odoo/addons/base/tests/test_db_cursor.py

```diff
@@ -31,6 +33,15 @@ def test_execute_bad_params(self):
             with self.assertRaises(ValueError):
                 cr.execute("SELECT id FROM res_users WHERE id=%s", '1')
 
+    @mute_logger('odoo.sql_db')
+    def test_normal_cursor(self):
+        with registry().cursor() as cr:
+            cr.close()
+            with self.assertRaises(psycopg2.InterfaceError):
+                cr.execute("SELECT 1")
+        with registry().cursor() as cr:
+            cr.close()
+            cr.close()
```

Comment:
I assume this is in order to test whether closing a closed cursor works?

Documenting the purpose of the test (and probably having two tests since we're testing two things) would be a good idea, because `test_normal_cursor` doesn't say much that's useful.

And the test class is actively misleading, it should be renamed and the docstring removed.

And this specific test should probably do without a `with`, as the previous section already tests what happens on committing a closed cursor, no?

Reply:
I slit both tests, rename the class and remove the useless mute_logger.

Reply:
> I slit both tests

A bit violent but ok.